--- share/server/core/classes/GlobalMainCfg.php.orig	2013-08-09 17:37:10.000000000 -0400
+++ share/server/core/classes/GlobalMainCfg.php	2013-10-07 16:44:13.608762311 -0400
@@ -47,6 +47,36 @@
      */
     public function __construct() {
         $this->validConfig = Array(
+            'authmysqldb'=> Array(
+                'host' => Array('must' => 1,
+                    'editable' => 1,
+                    'default' => 'localhost',
+                    'match' => MATCH_STRING),
+                'port' =>  Array('must' => 1,
+                    'editable' => 1,
+                    'default' => 3306,
+                    'match' => MATCH_INTEGER),
+                'user' => Array('must' => 1,
+                    'editable' => 1,
+                    'default' => 'nagvis',
+                    'match' => MATCH_STRING),
+                'password' => Array('must' => 1,
+                    'editable' => 1,
+                    'default' => 'manager',
+                    'match' => MATCH_STRING),
+                'database' => Array('must' => 1,
+                    'editable' => 1,
+                    'default' => 'nagvis',
+                    'match' => MATCH_STRING),
+                'centreondatabase' => Array('must' => 1,
+                    'editable' => 1,
+                    'default' => 'centreon',
+                    'match' => MATCH_STRING),
+                'authlog' => Array('must' => 1,
+                    'editable' => 1,
+                    'default' => '/tmp/cn.log',
+                    'match' => MATCH_STRING)
+            ),
             'global' => Array(
                 'audit_log' => Array(
                     'must'     => 0,
